iblvideo/run.py

```diff
@@ -1,47 +1,62 @@
-from datetime import datetime
+import logging
+
 from oneibl.one import ONE
 from oneibl.patcher import FTPPatcher
-from .choiceworld import dlc
-from .weights import download_weights
-from . import __version__
+from iblvideo.choiceworld import dlc
+from iblvideo.weights import download_weights
+from iblvideo import __version__
+from ibllib.pipes import tasks
+
+_logger = logging.getLogger('ibllib')
+
+
+# re-using the Task class allows to not re-write all the logging, error management
+# and automatic settings of task statuses in the database
+class TaskDLC(tasks.Task):
+    gpu = 1
+    cpu = 4
+    io_charge = 90
+    level = 0
 
+    def _run(self, version=__version__):
+        # Download weights into ONE Cache directory under 'resources/DLC' if not exist
+        path_dlc = download_weights(version=version)
+        files_mp4 = list(self.session_path.joinpath('raw_video_data').glob('*.mp4'))
+        if len(files_mp4) == 0:
+            raise FileNotFoundError('no video file found')
+        # Run dlc on all three videos and upload data, set status to complete
```

Comment:
So now we do not account for the situation where there is less than three videos. Is that ok? Should we somehow log when there are videos missing?